NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -151,43 +151,69 @@ class NeuroID private constructor(
         return true
     }
 
-    fun setUserID(userId: String): Boolean {
-
-        val validUserID = this.validateUserId(userId)
-        if (!validUserID) {
-            return false
+    fun setRegisteredUserId(registeredUserId: String): Boolean {
```

Comment:
can we do `setRegisteredUserID`? trying to keep the function names consistent across all 3 sdks

Reply:
fixed. thanks!